docs/sink_connector.rst

```diff
@@ -0,0 +1,139 @@
+JDBC Sink Connector
+===================
+
+The JDBC sink connector allows you to export data from Kafka topics to any relational database with a JDBC driver.
+By using JDBC, this connector can support a wide variety of databases without requiring custom code for each one.
+The connector polls data from Kafka to write to the database, based on the topics subscription.
+It is possible to achieve idempotent writes with upserts.
+Auto-creation of tables, and limited auto-evolution is also supported.
+
+Quickstart
+----------
+
+To see the basic functionality of the connector, we'll be copying from a single topic to a local SQLite database.
```

Comment:
Maybe mention Avro data here to setup the Schema Registry requirement in the next sentence.